hr_attendance_management/models/hr_attendance_day.py

```diff
@@ -303,7 +358,6 @@ def extend_longest_break(extension_duration):
                 })
 
             att_break.write({
-                'system_modified': True,
```

Comment:
Are you sure you want to remove this?

Reply:
Yes it is a compute field depends on additional_duration